### PR TITLE
Port bridges summary

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-5921
Resolves L2B-6862

INB4:
- bridges-summary table's filters are not affecting TVL chart (as was before)
- ~~chart func hasn't been reused, we have to do cleaning across different tvl-related functions to get rid of excessive boilerplate ;p - will try to mess around with context to make it a bit more reusable~~

### CODE REVIEW

File: packages/frontend2/src/app/_components/chart/tvl/scaling-summary-tvl-chart.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }
+  }, [entries, excludeAssociatedTokens, includeFilter, filters])
+
+  const scalingSummaryQuery = api.scaling.summary.chart.useQuery({
+    range: timeRange,
+    excludeAssociatedTokens,
+    ...chartDataType,
+  })
+
+  const { data } = scalingSummaryQuery
```
Comment (torztomasz) [inline packages/frontend2/src/app/_components/chart/tvl/scaling-summary-tvl-chart.tsx:57]:
```suggestion
  const { data } = api.scaling.summary.chart.useQuery({
    range: timeRange,
    excludeAssociatedTokens,
    ...chartDataType,
  })
```

**Review by torztomasz — CHANGES_REQUESTED** [review]

File: packages/frontend2/src/app/_components/chart/tvl/scaling-summary-tvl-chart.tsx (hunk 1 above)
Comment (sdlyy) [reply packages/frontend2/src/app/_components/chart/tvl/scaling-summary-tvl-chart.tsx]: https://github.com/l2beat/l2beat/pull/4695/commits/51abc08a0ffac1f48f46b88930f3914bc4d4b1dd

File: packages/frontend2/src/app/(new)/(other)/bridges/summary/_components/table/bridges-summary-tables.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    () => ({
+      getCoreRowModel: getCoreRowModel(),
+      getSortedRowModel: getSortedRowModel(),
+      manualFiltering: true,
+      initialState: {
+        sorting: [
+          {
+            id: '#',
+            desc: false,
+          },
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend2/src/app/(new)/(other)/bridges/summary/_components/table/bridges-summary-tables.tsx]: https://github.com/l2beat/l2beat/pull/4695/commits/1218c70615a39a018e0dbc2c40b028d47888e63c

File: packages/frontend2/src/app/(new)/(other)/bridges/summary/_components/table/columns.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        height={18}
+        alt={`${ctx.row.original.name} logo`}
+      />
+    ),
+    size: 26,
+    meta: {
+      headClassName: 'w-0',
+      cellClassName: 'lg:!pr-0 pl-2',
+    },
+  }),
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend2/src/app/(new)/(other)/bridges/summary/_components/table/columns.tsx]: https://github.com/l2beat/l2beat/pull/4695/commits/79cad77c3dd321e382bbe9f0ce4eef88d22d3923

**Review by torztomasz — APPROVED** [review]

**Pull request merged**